### ISSUE TITLE
Razer Synapse (3) will not launch!

### ISSUE THREAD

**Issue by Zac08** [Whisky-App/Whisky#884]
### Description

Here I go again. Just installed Whisky today, and it's great. Some major flaws though. Not supporting windows 11 will be a red flag soon, and some apps just not opening in general. In my case, it is the mouse app "Razer Synapse 3".  Tried to open it today and there we something strange going on. The app was refusing to open. It flashed (something) for a second in the dock, but then vanished and whisky finished the script. Is this normal?

### Steps to reproduce

Reproduce the problem, or reproduce the solution?
Download razer synapse 3 at there download page > https://www.razer.com/ap-en/synapse-3
Open Whisky > Create new bottle > Open c: folder > add "Razer Synapse 3" > then press 'Run'

https://github.com/Whisky-App/Whisky/assets/145438617/053313f9-a2ea-4339-aa3f-b5040025247c

That's what it looks like. ^^

### Expected behaviour

I was expecting for the app to launch as it had signs like: The Whisky logo on the exe, The fact that it was actually supported by whisky but still doesn't launch. Please help guys, this is the only oppurtunity ive had to change my mouses software since I got it 2 years ago :(

### Logs

```bash
Not exactly sure which logs to look for as they all say things like "Mysync is up and running, and tasks were successfully completed". 

fyi, I do have my mouse plugged in
```

### What version of Whisky are you using?

2.2.3

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [X] Yes my issue is written in English

**Comment by nezlobnyj** [comment]
@Zac08 I use Razer mouse with Sonoma, so researched firmware upgrade too. What I've gathered is Synapse won't work fully neither via Wine, nor in Win VMs. Synapse will start in both, but will not detect the hardware and is basically unusable.

Now if you only want to update the firmware - there are hardware-specific executables from Razer you can use, those work, but in VM only for me.

E.g., I have DeathAdder v3 Pro, so searched for "DAv3 Pro firmware" and got to this link - https://mysupport.razer.com/app/answers/detail/a_id/6403/~/razer-deathadder-v3-pro-firmware-updater-%7C-rz01-04630. There is a shortened URL to actual firmware tool, which was downloaded and opened in Windows 11 VM and following the steps provided I updated to the latest dongle and mouse firmware.

For VM - you can download trial of Parallels and install Win 11 ARM (not activated) for free. This will provide you enough time and software to upgrade.

Hope this helps.

TL;DR - not a bug, will never work, use other tools :)
